Dockerfile

```diff
@@ -13,9 +13,10 @@ COPY kubectl /usr/local/bin/kubectl
 RUN yum install -y epel-release 
 RUN yum install -y ansible dmidecode which python-pip pciutils
 RUN pip3 install --upgrade pip
-RUN pip3 install elasticsearch
+RUN pip3 install elasticsearch-dsl
```

Comment:
i know dsl is built on top of elasticsearch library, but can you confirm that we won't need es here @dry923

Reply:
I built an image with that docker file and it worked fine.

Reply:
ack, sounds good